docs/content/docs/reference/hasura-auth/schema.mdx

```diff
@@ -0,0 +1,85 @@
+---
+title: 'Schema'
+---
+
+Hasura Auth stores all its data in a dedicated `auth` PostgreSQL schema. It is created or updated when Hasura Auth starts.
```

Comment:
Is the `auth` schema really created by Hausra on startup?

If I remember correctly, Haura Auth relies on the `auth` scheme already being there. Hasura Auth then only manages its DB migrations and Haura metadata.

Reply:
Good point: the schema must be created beforehand, while its contents is migrated on startup. Will rephrase the above.